build.sbt

```diff
@@ -23,9 +22,9 @@ lazy val chronoscala = (project in file("."))
 
     publishMavenStyle := true,
 
-    scalaVersion := scala210,
+    scalaVersion := "2.11.12",
 
-    crossScalaVersions := Seq(scala210, "2.11.12", "2.12.10", "2.13.1"),
+    crossScalaVersions := Seq("2.11.12", "2.12.10", "2.13.1"),
```

Comment:
The current latest version is as follows (2020/11/24)
cf. https://www.scala-lang.org/download/
- `2.11.12`
- `2.12.12`
- `2.13.4`